go/v1beta1/main/main.go

```diff
@@ -0,0 +1,22 @@
+package main
```

Comment:
This is for use in a future PR where code in this repo will be built and run in the Dockerfile.